Fig6_PyGMT_geopandas.py

```diff
@@ -0,0 +1,148 @@
+import geopandas as gpd
+import pygmt
+
+provider = "https://naciscdn.org/naturalearth"
+states = gpd.read_file(f"{provider}/110m/cultural/ne_110m_admin_1_states_provinces.zip")
+states["area_sqkm"] = states.geometry.to_crs(epsg=6933).area / 10 ** 9
+states = states[states["name"] != "Alaska"]
+rivers = gpd.read_file(f"{provider}/50m/physical/ne_50m_rivers_lake_centerlines.zip")
+rivers = rivers.cx[-126:-66, 24.9:49.1]
+cities = gpd.read_file(f"{provider}/110m/cultural/ne_110m_populated_places_simple.zip")
+cities = cities.cx[-126:-66, 24.9:49.1]
+cities_large = cities[cities["worldcity"] == 1].copy()
+cities_large = cities_large.cx[-126:-66, 24.8:43]
+cities_small = cities[cities["worldcity"] != 1].copy()
+cities_small = cities_small.cx[-126:-66, 26:45]

```

Comment:
No need to distinguish large and small cities in this case, right?

Reply:
No, there is no overlapping of the labels of the cities.